dashboard/app/controllers/projects_controller.rb

```diff
@@ -599,6 +600,14 @@ def limited_gallery?
     return redirect_to "/projects/#{params[:key]}/#{params[:channel_id]}/view" if !is_owner && request.path.ends_with?('/edit')
   end
 
+  private def set_lab2_responsive_view_options
+    return nil unless @level.uses_lab2?
+  
+    # If the user is on the play view '/projects/channel_id', set `response_content`.`
+    if params[:share] == true
```

Comment:
This is checked in `application.html.haml` [here](https://github.com/code-dot-org/code-dot-org/blob/4923cc336bdec49e275bc235dcb13087d9aa6139/dashboard/app/views/layouts/application.html.haml#L16).

Reply:
Wondering if we should technically use the `sharing` value computed on line 374 so that we're consistent with what we pass into `level_view_options`. It shouldn't matter for now since we don't do anything special for embedded projects in lab2 currently, but just to be safe? Curious if @breville has thoughts

Reply:
Yeah - I was wondering about this earlier. Thanks! Will wait on Brendan to weigh in.

Reply:
Discussed offline - I'll go ahead and update to use `sharing`.